### ISSUE TITLE
Consider restructuring to move "Developer features" and convert "Features" into "Products"

### ISSUE THREAD

**Issue by neynah** [sandstorm-io/sandstorm-website#234]
As a result of changes in #233, our "Features" page will now have subpages for "Sandstorm Standard" features, "Sandstorm for Work" features, and also "Developer" features. "Developer" features now seems a bit out of place since the other subpages are dedicated to user-facing product features:

![image](https://cloud.githubusercontent.com/assets/855341/18766355/b57d789a-80ce-11e6-890e-1812b03e290d.png)

In the future I think we should:

1. Consider moving all Developer features content into docs or onto it's own separate page on our website. Although it's important to communicate why developers should develop on Sandstorm, I suspect Developer features just doesn't quite fit into the customer's mental model when they're shopping for Sandstorm. @zarvox and I think the main purpose of our Features/Products page on our website should be to communicate our user/admin-facing product features.

2. If we move [Developer features](https://sandstorm.io/developer) elsewhere then we can could rename "Features" to "Products"; this page could then list our 3 product options: "Sandstorm Standard", "Sandstorm on Oasis", and "Sandstorm for Work". Then we would have clean links between product options on /get to the Products section of our website where users can learn more about each of the products in greater depth.

3. Consider removing "Code" and "How it Works" from the nav bar and either move the content into docs and/or link to them from the footer & from other places on our website.

4. Change pages linked in our nav bar at smaller widths. In "tablet" view we emphasize "Features", "How it Works", "Blog", and "About". In mobile view we emphasize "How it Works" and "About", and in more extreme cases just "How it Works". I think we should enable users to easily navigate to more product & information focused pages. For mobile I propose: "Get Sandstorm" and "Blog". For tablet I propose: "Get Sandstorm", "Features, "Blog", and "Docs". I'm open to other suggestions! 

![image](https://cloud.githubusercontent.com/assets/855341/18766584/cf15dfb2-80cf-11e6-8393-d0f34f4d7540.png)

The rationale behind these changes is to achieve better information architecture that better maps to the user's mental model to aid them in becoming more informed about Sandstorm & its offerings as well as make a decision about using/purchasing Sandstorm.

**Comment by kentonv** [comment]
IIRC we decided that it wasn't necessarily for "Get Sandstorm" to be shown in the mobile view because there's already a prominent button linking there from the top of the front page.

While I agree that "How it Works" is likely not important to most customers, I think it is very important to a certain segment of our audience that are likely to be influencers. Therefore I do not want to remove it from the nav, although I am OK with changing which things we show on mobile.

FWIW, the standard thing to do here is use a hamburger menu on mobile. Not that I like hamburger menus, but I think our strategy of dropping things from the top bar is unusual.

**Comment by neynah** [comment]
>FWIW, the standard thing to do here is use a hamburger menu on mobile. Not that I like hamburger menus, but I think our strategy of dropping things from the top bar is unusual.

Yeah, I think what we currently have is worse than having a hamburger menu with links to all our pages. I opened a new issue at #235.
